ctx.go

```diff
@@ -817,6 +817,21 @@ func (c *DefaultCtx) Locals(key any, value ...any) any {
 	return value[0]
 }
 
+// Locals function utilizing Go's generics feature.
+// This function allows for manipulating and retrieving local values within a request context with a more specific data type.
+func Locals[V any](c Ctx, key any, value ...any) V {
+	ctx, ok := c.(*DefaultCtx)
```

Comment:
what is the need for the type assertion?

Reply:
I realize we shouldn't use a type assertion here because we don't need access to private functions.

Reply:
pls update this